cachito/workers/pkg_manager.py

```diff
@@ -148,31 +162,17 @@ def add_deps_to_bundle(src_deps_path, dest_cache_path, request_id):
         content of src_deps_path to
     :param int request_id: the request the bundle is for
     """
-    config = get_worker_config()
-    deps_path = os.path.join(config.cachito_bundles_dir, 'temp', str(request_id), 'deps')
+    bundle_dir = get_request_bundle_dir(request_id)
```

Comment:
Optional: You can skip creating this variable here and directly call the method in the next statement.

Reply:
I did it the other way for readability, but I'll change it.